src/Dapr.Client/DaprClient.cs

```diff
@@ -939,6 +940,293 @@ public async Task<StateEntry<TValue>> GetStateEntryAsync<TValue>(string storeNam
             string id,
             CancellationToken cancellationToken = default);
 
+        #region Cryptography
+
+        /// <summary>
+        /// Encrypts an array of bytes using the Dapr Cryptography encryption functionality.
+        /// </summary>
+        /// <param name="vaultResourceName">The name of the vault resource used by the operation.</param>
+        /// <param name="plainTextBytes">The bytes of the plaintext value to encrypt.</param>
+        /// <param name="algorithm">The name of the algorithm used to wrap the encryption key.</param>
+        /// <param name="keyName">The name of the key to use from the Vault for the encryption operation.</param>
+        /// <param name="dataEncryptionCipher">The name of the cipher to use for the encryption operation.</param>
+        /// <param name="cancellationToken">A <see cref="CancellationToken"/> that can be used to cancel the operation.</param>
+        /// <returns>An array of encrypted bytes.</returns>
+        [Obsolete("The API is currently not stable as it is in the Alpha stage. This attribute will be removed once it is stable.")]
+        public abstract Task<byte[]> EncryptAsync(string vaultResourceName, byte[] plainTextBytes,
```

Comment:
Is "plaintext" intended in the "common term for arbitrary bytes to be encrypted" sense or is there an expectation that the value actually represent text?  If the former, perhaps a different argument name (e.g. `source` or `sourceBytes` or even `bytesToBeEncrypted`)?

Reply:
I think I added the byte[] overload after the string variant, so it was just a renaming of the existing parameter ("plaintext") to make it match the type. That said, yes, "ciphertext" is a commonly used term in cryptography to describe the encrypted version of data and "plaintext" is the original or since-decrypted version of that data. The two are the common opposite of one another, so it would make sense to refer to this just as "plaintext" or "plaintextBytes" (the T shouldn't have been capitalized) with a clarifying description in the XML to make it clear to the developer that these are the bytes to be encrypted.

Reply:
Updated all uses of "plainText" and "cipherText" to "plaintext" and "ciphertext" for consistency.